server/src/main/java/org/opensearch/search/aggregations/bucket/missing/MissingOrder.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search.aggregations.bucket.missing;
+
+import org.opensearch.common.inject.Provider;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
+
+import java.io.IOException;
+import java.util.Locale;
+
+/**
+ * Composite Aggregation Missing bucket order.
+ */
+public enum MissingOrder implements Writeable {
```

Comment:
Suggestion:

 to ensure consistency and avoid inadvertent typos/string changes let's include a static `NAME` variable.

`public static final String NAME = "missing_order";`

I know this doesn't exist for `missing_bucket` (yet) but I think it would be good to capture these parameter strings in variables instead of string literals to help avoid oopsie typos.

Reply:
Totally agree! Done.